reference/7.2/Microsoft.PowerShell.Core/About/about_PowerShell_Config.md

```diff
@@ -39,358 +72,380 @@ installation.
 ### CurrentUser (per-user) configurations
 
 You can also configure PowerShell on a per-user basis by placing the file in
-the user-scope configuration directory. The user configuration directory can
-be found across platforms with the command
+the user-scope configuration directory. The user configuration directory can be
+found across platforms with the command
 `Split-Path $PROFILE.CurrentUserCurrentHost`.
 
-## General configuration settings
+### Scope precedence
 
-### ExecutionPolicy
+On Windows, settings managed by Windows Group Policy take precedence over
+settings in the configuration file. Group Policy doesn't exist on non-Windows
+platforms.
 
-> [!IMPORTANT]
-> This configuration only applies on Windows platforms.
+After Group Policy, settings defined at the **AllUsers** level take precedence
+over settings defined for the **CurrentUser** level.
```

Comment:
This is an interesting inversion of typical setting merging, presumably for security concerns.